src/insert/spacefinder/article.ts

```diff
@@ -143,47 +100,6 @@ const decideAdditionalSizes = async (
 };
 
 const addDesktopInline1 = (fillSlot: FillAdSlot): Promise<boolean> => {
-	const tweakpoint = getCurrentTweakpoint();
-	const hasLeftCol = ['leftCol', 'wide'].includes(tweakpoint);
-
-	let ignoreList = `:scope > :not(p):not(h2):not(ul):not(.${adSlotContainerClass}):not(#sign-in-gate):not(.sfdebug)`;
-	if (hasLeftCol) {
-		ignoreList +=
-			':not([data-spacefinder-role="richLink"]):not([data-spacefinder-role="thumbnail"])';
-	}
-
-	const isImmersive = window.guardian.config.page.isImmersive;
-
-	const rules: SpacefinderRules = {
-		bodySelector: articleBodySelector,
-		candidateSelector: ':scope > p, [data-spacefinder-role="nested"] > p',
-		minAbove: isImmersive ? 700 : 300,
-		minBelow: 300,
-		opponentSelectorRules: {
-			// don't place ads right after a heading
-			':scope > h2, [data-spacefinder-role="nested"] > h2': {
-				minAboveSlot: isInHighValueSection ? 150 : 5,
-				minBelowSlot: isInHighValueSection ? 0 : 190,
-			},
-			[`.${adSlotContainerClass}`]: {
-				minAboveSlot: 500,
-				minBelowSlot: 500,
-			},
-			[ignoreList]: {
-				minAboveSlot: 35,
-				minBelowSlot: 400,
-			},
-			'[data-spacefinder-role="immersive"]': {
-				minAboveSlot: 0,
-				minBelowSlot: 600,
-			},
-			'figure.element--supporting': {
-				minAboveSlot: 100,
```

Comment:
The same for this one which I can see it is one of the `inlineOpponentSelector` values but it has different `minAbove` and `minBelow` than what we have in the old code.

Reply:
I found it odd that supporting elements have their own rule (something to be investigated I think), but you're right I'll add this back in.

Reply:
I believe if it needs investigation then like you said we can have it back and probably there will be another iteration to drop the rules that we think we might not need it.